src/DynamoCore/Library/LibraryCustomization.cs

```diff
@@ -10,27 +10,55 @@
 
 namespace Dynamo.Engine
 {
-    internal interface ILibraryCustomizationServices
+    /// <summary>
+    /// Provides access to library custoization objects
+    /// </summary>
+    public interface ILibraryCustomizationServices
     {
+        /// <summary>
+        /// Provides a LibraryCustomization for a given assembly
+        /// </summary>
+        /// <param name="assemblyPath">Assembly for which library customization is needed</param>
+        /// <returns></returns>
```

Comment:
Being nit-picky today... :)

```csharp
/// <summary>
/// Call this method to obtain the corresponding LibraryCustomization object
/// for a given assembly path.
/// </summary>
/// <param name="assemblyPath">Path of assembly for which LibraryCustomization
/// object is to be obtained.</param>
/// <returns>Returns the corresponding LibraryCustomization object if one is 
/// found, or null otherwise.</returns>
```